### ISSUE TITLE
[BUG] Performance monitor app does not appear in Android auto

### ISSUE THREAD

**Issue by UbayGD** [jilleb/mqb-pm#154]
Hello!
I am having trouble getting the app to add correctly to Android Auto but have not been successful. My car is a 2020 Skoda Fabia.

I think I have followed the steps correctly:
1 - Enable developer mode in Android Auto and unknown sources.
2 - Install VAG Extensions for Android Auto (from AAAD)
3 - Install Performance Monitor for AA (from AAAD)

By following these steps the only thing I have managed to do is to display a "Performance Monitor Settings" icon. Just in case, I also rooted the phone and installed an application to enable third-party apps on Android Auto, without success.
I don't know what else I can try. I don't think I'm missing anything.

Here I leave information about the AA app version and firmware in my car.

Android Auto: 7.1
Car software: 8742

I'm sorry that the images are in Spanish but I think you can understand them.
![image](https://user-images.githubusercontent.com/10211264/151527607-0f959d1c-2739-47ee-9750-a893822b9c25.png)
![image](https://user-images.githubusercontent.com/10211264/151527654-4963441d-4e2b-4981-927c-fa1414c1b6a3.png)
![image](https://user-images.githubusercontent.com/10211264/151527701-500214eb-d5bf-4662-a137-d9241e99bbae.png)

Greetings and thanks! :)

**Comment by ale99viv** [comment]
did you find a solution?

**Comment by Maverick-cmd** [comment]
This is for older versions of Android Auto, before version 3, current versions are version 7.x.xxxxxx-release. Also versions for other things are way out of date as well. It will not work unless you downgrade or reverse engineer and rewrite some code.

**Comment by ale99viv** [comment]
I currently use Android Auto version 7.x.xxxxxx and since today Performance Auto start working perfectly on my skoda fabia 2019. I don't know the reason but it works. I have installed Performance Monitor 0.88.0 through AAStore

**Comment by Maverick-cmd** [comment]
Interesting, my attempt was way more difficult. Everything works for you though?

**Comment by ale99viv** [comment]
Yes everything works fine. I started seeing the indicators but they didn't work and after about 2 weeks they started working by themselves
